src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss

```diff
@@ -0,0 +1,19 @@
+journal-early-start-modal {
+  @extend .modal-alert-wrapper;
+
+  .modal-alert {
+    width: 500px;
+  }
+
+  .modal-alert-message {
+    margin-bottom: 25px;
```

Comment:
24px would be a better representation for our grid

Reply:
pushing this change now